### DISCUSSION TITLE
Confusion about domain and dns records

### DISCUSSION THREAD

**Discussion by samueldovi** [dokku/dokku#4410] [Q&A] [answered]
Hello.

so I did read the documentation about domains and dns setup configuration.
But I'm still kinda lost
What should my dns point to ? the ip of the app ? 
because I setup a domain  for the app and it does not deployed on the old IP
i got 3 app I want to use with those domains
api.mydomain.com
storage.mydomain.com
and the root for www.mydomain.com

should I set the domain of the apps to those ? but after what should I set the A records to ?
I bought my domain at domain.com if that help.

sorry I'm a total noob to webdev.

**Comment by josegonzalez** [comment, marked as answer]
You should set your root domain name as an "A record" for the IP address of your server. For subdomains, you can set them as a CNAME to that initial A record, or you can have them as an A record to the IP address of your server.

**Reply by samueldovi** [reply]
It worked thanks !
It's a little clearer now.
